.github/workflows/release.yml

```diff
@@ -61,8 +61,8 @@ jobs:
           tag: ${{ github.ref }}
 
   dockerfile:
```

Comment:
This looks a bit redundant to what's happening in `.github/workflows/request-docker-release.yml`, but I guess there is a good reason?

Reply:
Redundant yes. One could maybe use a workflow dispatch, but this currently works fine and keeps all the information in the same place.

Maybe we need to refactor at some point by creating custom actions.